bin/libsw.cmd

```diff
@@ -21,7 +21,19 @@ if not exist "%SPARK_HOME%/bin/spark-submit.cmd" (
 )
 exit /b 0
 
+
+:checkJava
+
+set "JV="
+For /F "Tokens=3" %%A In ('java -version 2^>^&1') Do If Not Defined JV Set "JV=%%~A"
```

Comment:
`For` or `for` ?

Reply:
good catch! Cmd is not case sensitive, but I will turn these all to lower case to be consistent